viewer/src/components/boards/Resources.vue

```diff
@@ -146,6 +146,21 @@
             <Coal :pieceId="-1" :targetState="{ x: 858, y: 57 }" :canClick="false" :transparent="true" :scale="0.2" />
         </template>
 
+        <template v-if="isMiddleEast">
```

Comment:
This is okay, but maybe consider changing it to how it is in USA recharged (add actual pieces of oil).

Reply:
Thanks, I thought about that, but I'm not sure if there's enough space to add the pieces individually. I'd rather leave it like this for now.